### PR TITLE
1st PR for the site

### PR DESCRIPTION
**Pull request by zinint**
@yugoslavskiy please review and correct or approve the changes. When we'll finish with the EN one, I'll Immediately do the RU one.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>OSCD</title>
-    <link rel="stylesheet" href="./hack.css">
+    <link rel="stylesheet" href="./hack_black.css">
```
Comment (yugoslavskiy) [inline index.html]: I think there is no need to change the home page styles at all mate (: let it be more like a usual normal website, and change styles only in sprints-related pages.

File: index.html [hunk 2]
```diff
@@ -44,53 +44,91 @@ <h2>Who</h2>
       We are cybersecurity enthusiasts. If you share the idea of global collaboration to combat threats, you are very welcome to join the next sprint.
       <br><br>
       <h2>When</h2>
-      The next two weeks-long sprint will start <a href="./sprints/sprint_2.html">October 5, 2020</a>.
+      The second two weeks-long sprint ended in <a href="./sprints/sprint_2.html">October 2020</a>.
```
Comment (yugoslavskiy) [inline index.html]: It's a good idea to mention the previous sprint!
Let's use the "previous" and "next" words (not 3rd/2nd), as readers don't really need to know the number here.

> The third two weeks-long sprint will start <a href="./sprints/sprint_3.html">Fall 2021</a>.

I would reuse [our former version](https://github.com/oscd-initiative/oscd/blob/4612cf1abb139a55cf76155864ad48a81a8a8865/index.html#L41):

> The next sprint is planned for Winter 2021. Details will be provided on our <a href="https://twitter.com/oscd_initiative">twitter</a> soon.

File: index.html [hunk 3]
```diff
@@ -44,53 +44,91 @@ <h2>Who</h2>
...
-      - Gleb Sukhodolskiy (<a href="https://www.angaratech.ru/">Angara technologies group</a>) 🇷🇺 <br>
-      - Victor Sergeev, <a href="https://twitter.com/stvetro">@stvetro</a> (<a href="https://www.helpag.com">Help AG</a>) 🇦🇪 <br>
-      - Ilyas Ochkov, <a href="https://t.me/CatSchrodinger">@CatSchrodinger</a> (Independent Researcher) 🇷🇺 <br>
-      - James Pemberton, <a href="https://twitter.com/4A616D6573">@4A616D6573</a> (<a href="https://www.hydro.com.au">Hydro Tasmania</a>) 🇦🇺 <br>
-      - Denis Beyu (<a href="https://citto.ru">GKU TO CITTO</a>) 🇷🇺 <br>
-      - Mateusz Wydra, <a href="https://twitter.com/sn0w0tter">@sn0w0tter</a> (<a href="https://www.relativity.com/ediscovery-software/relativityone/trust/threat-prevention/our-team/">Relativity</a>) 🇵🇱 <br>
-      - Jakob Weinzettl, <a href="https://twitter.com/mrblacyk">@mrblacyk</a> (<a href="https://www.tieto.com/en/what-we-do/managed-services-and-integration/security-services/security-operations-center/">Tieto SOC</a>) 🇵🇱 <br>
+      <h2>Previous sprints participants</h2>
+      <h4>Every sprint you earn one &#11088</h4>
```
Comment (yugoslavskiy) [inline index.html]: Stars look great! 
Could you please also put companies/"independent researcher" descriptions for all the members as we did it before?

Comment (yugoslavskiy) [reply index.html]: I think it also makes sense to sort the participants by the amount of stars in the list.

File: index.html [hunk 4]
```diff
@@ -44,53 +44,91 @@ <h2>Who</h2>
...
       - <a href="https://2019.hack.lu">hack.lu</a> organizers for a place on conference premises during the Sprint #1<br>
-      - Andrey <a href="https://www.behance.net/PolarLetters">PolarLetters</a> for the logo
+      - Andrey <a href="https://www.behance.net/PolarLetters">PolarLetters</a> for the logo<br>
+      - <a href="https://twitter.com/JohnLaTwC">John Lambert</a> for continuos support<br>
       <br><br>
       <h2>Contacts</h2>
-      - <a href="https://twitter.com/oscd_initiative">Twitter</a><br>
-      - <a href="https://join.slack.com/t/oscd/shared_invite/enQtODc1NjA0MzU0NTQ4LWMyMzdkNTJkZDUzZWZkNDk5M2JlNGY0MWRkMDlmMjg3NjBmZjBiMTg0MGRmMzA1YzI4ZWM3ZjZlMGQ3NWMwYzM">Slack</a> 
+      - <a href="https://twitter.com/oscd_initiative">OSCD Twitter</a><br>
+      - <a href="https://twitter.com/i/lists/1308954210470428672">OSCD Participants</a>
```
Comment (yugoslavskiy) [inline index.html]: I would say "OSCD Participants Twitter list"

File: sprints/sprint_1.html [hunk 5]
```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>OSCD</title>
-    <link rel="stylesheet" href="../hack.css">
+    <link rel="stylesheet" href="../hack_black.css">
```
Comment (yugoslavskiy) [inline sprints/sprint_1.html]: let's left sprint stiles as they are %) better create a new stile for the new sprint and not apply it for the others

File: sprints/sprint_3.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+      <br>
+      <center> <a href="sprint_2.html">EN</a> | <a href="sprint_2_ru.html">RU</a> </center>
+      <div>
+      <p align="center">
+        <img src="../img/oscd_logo_v2.gif" alt="OSCD">
+      </p>
+      </div>
+      <center> <p style="font-size: 16px;">OSCD Sprint #3 | Coming soon..</p> </center>
+      <br>
+      It's been a while since <a href="sprint_2.html" target="_blank">the Second OSCD Sprint</a>. We are currently working on the Third one. As always we will focus on multiple areas of practical CyberSecurity. You will soon be able to find more information as well as a detailed backlog, etc., here.
```
Comment (yugoslavskiy) [inline sprints/sprint_3.html]: I would rather use [our former form](https://github.com/oscd-initiative/oscd/blob/4612cf1abb139a55cf76155864ad48a81a8a8865/sprints/sprint_2.html):

> Details will follow soon. Subscribe to the <a href="https://twitter.com/oscd_initiative">twitter</a> for updates.

File: sprints/sprint_3.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+      <br>
+      <center> <a href="../index.html">Home</a> </center>
+      <br>
+      <center> <a href="sprint_2.html">EN</a> | <a href="sprint_2_ru.html">RU</a> </center>
+      <div>
+      <p align="center">
+        <img src="../img/oscd_logo_v2.gif" alt="OSCD">
+      </p>
+      </div>
+      <center> <p style="font-size: 16px;">OSCD Sprint #3 | Coming soon..</p> </center>
```
Comment (yugoslavskiy) [inline sprints/sprint_3.html]: you can remove this and follow [previous example](https://github.com/oscd-initiative/oscd/blob/4612cf1abb139a55cf76155864ad48a81a8a8865/sprints/sprint_2.html)

**Comment by yugoslavskiy** [comment]
@zinint hey hey! sorry for the delay, mate. please let me know what do you think about the comments

File: index.html (hunk 1 above)
Comment (zinint) [reply index.html]: I just think that `hack_clack.css` is cooler (: I'd change it (:

File: index.html (hunk 2 above)
Comment (zinint) [reply index.html]: I agree, thanks!

File: index.html (hunk 3 above)
Comment (zinint) [reply index.html]: > Could you please also put companies/"independent researcher" descriptions for all the members as we did it before?

It had been done. Am I missing smth?

> I think it also makes sense to sort the participants by the amount of stars in the list.

Again with the sorting? ((: Ok, I'll do it.

File: index.html (hunk 4 above)
Comment (zinint) [reply index.html]: Done, thanks!

File: sprints/sprint_1.html (hunk 5 above)
Comment (zinint) [reply sprints/sprint_1.html]: New style for every sprint is a good idea, but I'd still change the default one to `hack_clack.css`.

File: sprints/sprint_3.html (hunk 7 above)
Comment (zinint) [reply sprints/sprint_3.html]: Done, thanks!

File: sprints/sprint_3.html (hunk 6 above)
Comment (zinint) [reply sprints/sprint_3.html]: Done, thanks!

File: index.html (hunk 2 above)
Comment (zinint) [reply index.html]: https://github.com/oscd-initiative/oscd/pull/4/commits/9e639dffc3c8efb5ed756222da75a1408e508a8c

Comment (zinint) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (zinint) [reply index.html]: Also here https://github.com/oscd-initiative/oscd/pull/4#discussion_r650711910

File: index.html (hunk 3 above)
Comment (zinint) [reply index.html]: https://github.com/oscd-initiative/oscd/pull/4/commits/dc07dae46a9ae22f8837b26e8decfab8f0c3f4be

**Comment by zinint** [comment]
@yugoslavskiy yo! Thanks for your time! I’ve made changes. Please review and check the unresolved conversations!

File: sprints/sprint_1.html (hunk 5 above)
Comment (yugoslavskiy) [reply sprints/sprint_1.html]: then the main page will have a style of the second sprint %) let the main page be simple, neutral, and clean

File: index.html (hunk 1 above)
Comment (yugoslavskiy) [reply index.html]: answered here https://github.com/oscd-initiative/oscd/pull/4#discussion_r654148937
let's proceed with the conversation in this thread (:

I believe that the homepage should stay as simple and neutral as possible. Having a new style for each sprint is a good thing. I hope we will see more of them in the upcoming years.

If you strongly believe that it should not be this way — that's fine! Let's vote in the chat (:

File: index.html (hunk 3 above)
Comment (yugoslavskiy) [reply index.html]: > Every sprint you earn one

Could you please change it to something neutral yet informative, i.e.:

```
Stars reflect the number of sprints participated.
```

Current wording harshly pushes a reader to a position of a person that really needs the stars. Which is not always the case

File: index.html (hunk 1 above)
Comment (zinint) [reply index.html]: Guess what, I visited the site again and today I agree (:

File: index.html (hunk 3 above)
Comment (zinint) [reply index.html]: Done, thanks for the advice!

File: sprints/sprint_1.html (hunk 5 above)
Comment (zinint) [reply sprints/sprint_1.html]: Guess what, I visited the site again and today I agree (:

**Comment by zinint** [comment]
@yugoslavskiy yo! Thanks one more time! I made the changes again. Please review!

File: index.html [hunk 8]
```diff
@@ -20,9 +20,9 @@
     <link rel="icon" type="image/png" sizes="96x96" href="./img/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="./img/favicon-16x16.png">
     <link rel="manifest" href="./img/manifest.json">
-    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileColor" content="#000000">
```
Comment (yugoslavskiy) [inline index.html]: I googled the code and it seems that it is black %) could you please change it back to white as it is the home page?

File: index.html [hunk 9]
```diff
@@ -20,9 +20,9 @@
...
     <meta name="msapplication-TileImage" content="./img/ms-icon-144x144.png">
-    <meta name="theme-color" content="#ffffff">
+    <meta name="theme-color" content="#000000">
```
Comment (yugoslavskiy) [inline index.html]: I googled the code and it seems that it is black %) could you please change it back to white as it is the home page?

File: index.html [hunk 10]
```diff
@@ -44,53 +44,91 @@ <h2>Who</h2>
...
       <h2>Thanks to</h2>
       - <a href="https://2019.hack.lu">hack.lu</a> organizers for a place on conference premises during the Sprint #1<br>
-      - Andrey <a href="https://www.behance.net/PolarLetters">PolarLetters</a> for the logo
+      - Andrey <a href="https://www.behance.net/PolarLetters">PolarLetters</a> for the logo<br>
+      - <a href="https://twitter.com/JohnLaTwC">John Lambert</a> for continuos support<br>
       <br><br>
       <h2>Contacts</h2>
-      - <a href="https://twitter.com/oscd_initiative">Twitter</a><br>
-      - <a href="https://join.slack.com/t/oscd/shared_invite/enQtODc1NjA0MzU0NTQ4LWMyMzdkNTJkZDUzZWZkNDk5M2JlNGY0MWRkMDlmMjg3NjBmZjBiMTg0MGRmMzA1YzI4ZWM3ZjZlMGQ3NWMwYzM">Slack</a> 
+      - <a href="https://twitter.com/oscd_initiative">OSCD Twitter</a><br>
```
Comment (yugoslavskiy) [inline index.html]: i believe that there is no need to mention that it is "OSCD" twitter %) I would rather say "our" twitter

File: sprints/sprint_1.html [hunk 11]
```diff
@@ -20,9 +20,9 @@
     <link rel="icon" type="image/png" sizes="96x96" href="../img/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="../img/favicon-16x16.png">
     <link rel="manifest" href="../img/manifest.json">
-    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileColor" content="#000000">
```
Comment (yugoslavskiy) [inline sprints/sprint_1.html]: same as with the homepage

File: sprints/sprint_1.html [hunk 12]
```diff
@@ -20,9 +20,9 @@
...
     <meta name="msapplication-TileImage" content="../img/ms-icon-144x144.png">
-    <meta name="theme-color" content="#ffffff">
+    <meta name="theme-color" content="#000000">
```
Comment (yugoslavskiy) [inline sprints/sprint_1.html]: same as with the homepage

File: sprints/sprint_1.html [hunk 13]
```diff
@@ -64,38 +64,6 @@ <h2>Results</h2>
       Listing of the rules is available in the description of the <a href="https://github.com/Neo23x0/sigma/pull/554">Pull Request</a> to Sigma repo.<br>
       Summary published on <a href="https://medium.com/oscd/oscd-threat-detection-sprint-1-c42317e06771">Medium</a> (EN) and <a href="https://habr.com/ru/post/482642/">Habr</a> (RU).
       <br><br>
```
Comment (yugoslavskiy) [inline sprints/sprint_1.html]: I believe that you can leave the list of participants for the sprint here as well, as it is only listing the participants of the particular sprint. please do the same for the sprint №2 page

File: sprints/sprint_1_ru.html [hunk 14]
```diff
@@ -65,38 +65,6 @@ <h2>Результаты</h2>
       Листинг правил доступен в описанни <a href="https://github.com/Neo23x0/sigma/pull/554">Pull Request</a> в репозитории Sigma.<br>
       Итоги опубликованы в статьях на <a href="https://habr.com/ru/post/482642/">Хабре</a> (RU) и <a href="https://medium.com/oscd/oscd-threat-detection-sprint-1-c42317e06771">Medium</a> (EN).
       <br><br>
```
Comment (yugoslavskiy) [inline sprints/sprint_1_ru.html:67]: please update the russian page accordingly 🙏

File: sprints/sprint_2.html [hunk 15]
```diff
@@ -108,6 +108,11 @@ <h2>Incident Response</h2>
       <br>
       <!-- The coverage was built upon <a href="https://github.com/oscd-initiative/Cortex-Analyzers" target="_blank">our fork</a> of repo with TheHive Analyzers and Responders, that we tagged with RE&CT IDs. -->
       That's why we've decided to focus on the TheHive Responders development this time. A detailed backlog on Incident Response could be found <a href="sprint_2_backlog.html" target="_blank">here</a>.
+      <br><br>
+      <h2>Results</h2>
+      - 242 new <a href="https://github.com/SigmaHQ/sigma/pull/1362">Sigma</a> rules added and 305 updated; <br>
```
Comment (yugoslavskiy) [inline sprints/sprint_2.html]: need to mention the OTR PR as well: https://github.com/SigmaHQ/sigma/pull/1158

File: sprints/sprint_2_ru.html [hunk 16]
```diff
@@ -110,6 +110,11 @@ <h2>Incident Response</h2>
       <br>
       <!-- The coverage was built upon <a href="https://github.com/oscd-initiative/Cortex-Analyzers" target="_blank">our fork</a> of repo with TheHive Analyzers and Responders, that we tagged with RE&CT IDs. -->
       Вот почему мы решили сосредоточиться на разработке Модулей Реагирования платформы TheHive. Детальный список задач по данному направлению можно найти <a href="sprint_2_backlog_ru.html" target="_blank">здесь</a>.
+      <br><br>
+      <h2>Результаты</h2>
+      - 242 новых правила <a href="https://github.com/SigmaHQ/sigma/pull/1362">Sigma</a> и 305 правил обновлены; <br>
```
Comment (yugoslavskiy) [inline sprints/sprint_2_ru.html]: please update the russian page accordingly 🙏

File: sprints/sprint_3.html [hunk 17]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```
Comment (yugoslavskiy) [inline sprints/sprint_3.html:1]: need both en and ru versions. you can use the previous one as an example basically

**Comment by zinint** [comment]
Made the requested changes, please review again

File: index.html (hunk 8 above)
Comment (zinint) [reply index.html]: Done, thx

File: index.html (hunk 9 above)
Comment (zinint) [reply index.html]: Done, thx

File: index.html (hunk 10 above)
Comment (zinint) [reply index.html]: Done, thx

File: sprints/sprint_1.html (hunk 11 above)
Comment (zinint) [reply sprints/sprint_1.html]: Done, thx

File: sprints/sprint_1.html (hunk 12 above)
Comment (zinint) [reply sprints/sprint_1.html]: Done, thx

File: sprints/sprint_1.html (hunk 13 above)
Comment (zinint) [reply sprints/sprint_1.html]: Done, for both sprints, thx

File: sprints/sprint_1_ru.html (hunk 14 above)
Comment (zinint) [reply sprints/sprint_1_ru.html:67]: Done, thx

File: sprints/sprint_2.html (hunk 15 above)
Comment (zinint) [reply sprints/sprint_2.html]: Done, thx

File: sprints/sprint_2_ru.html (hunk 16 above)
Comment (zinint) [reply sprints/sprint_2_ru.html]: Done, thx

File: sprints/sprint_3.html (hunk 17 above)
Comment (zinint) [reply sprints/sprint_3.html:1]: Done, and also added appropriate links, thx

File: index_ru.html [hunk 18]
```diff
@@ -44,53 +44,94 @@ <h2>Кто</h2>
...
       <br><br>
       <h2>Когда</h2>
-      Следующий двухнедельный спринт начнется <a href="./sprints/sprint_2_ru.html">5 Октября 2020 года</a>.
+      Предыдущий спринт закончился в <a href="./sprints/sprint_2_ru.html"> октябре 2020 года</a>.
+      <br>
+      <a href="./sprints/sprint_3_ru.html">Следующий спринт</a> запланирован на зиму 2021 года. Следите за новостями в <a href="https://twitter.com/oscd_initiative">twitter</a>.
       <br><br>
       <h2>Где</h2>
-      Большинство участников подключаются к спринту удаленно. О возможности личного присутствия будет объявлено вместе с деталями о следующем спринте.
+      GitHub. Дополнительная информация будет опубликована позже.
```
Comment (yugoslavskiy) [inline index_ru.html]: я бы в соответствии с английской версией написал "Дополнительная информация будет опубликована позже, вместе с деталями о следующем спринте." но это конечно вкусовщина. мне не нравится что нет конкретики, обрывается просто на слове "опубликуем потом". сейчас мне кажется что стоит и про твиттер тут еще раз сказать (лишним не будет). но решать тебе

File: index_ru.html [hunk 19]
```diff
@@ -35,7 +35,7 @@
       </div>
       <center> <p style="font-size: 16px;"><b>Open Security Collaborative Development</b></p> </center>
       <br>
-      Открытая международная инициатива специалистов по компьютерной безопасности, направленная на решение общих проблем, распространение знаний и улучшение компьютерной безопасности в целом.
+      Открытое международное сообщество специалистов по компьютерной безопасности, созданное для решения общих проблем и распространения знаний.
```
Comment (yugoslavskiy) [inline index_ru.html]: инициатива)

File: index_ru.html [hunk 20]
```diff
@@ -44,53 +44,94 @@ <h2>Кто</h2>
...
-      - Якоб Вайнзетл, <a href="https://twitter.com/mrblacyk">@mrblacyk</a> (<a href="https://www.tieto.com/en/what-we-do/managed-services-and-integration/security-services/security-operations-center/">Tieto SOC</a>) 🇵🇱 <br>
-      - Том Керн (<a href="https://www.nil.com/en/security/security-operations-center-soc-building-services/">NIL SOC</a>) 🇸🇮 <br>
-      - Сергей Солдатов, <a href="https://twitter.com/SVSoldatov">@SVSoldatov</a> (<a href="https://www.kaspersky.com/enterprise-security/security-operations-center-soc">Kaspersky MDR</a>) 🇷🇺 <br>
-      - Иэн Дэвис (<a href="https://www.tieto.com/en/what-we-do/managed-services-and-integration/security-services/security-operations-center/">Tieto SOC</a>) 🇨🇿 <br>
-      - Даниил Югославский, <a href="https://twitter.com/yugoslavskiy">@yugoslavskiy</a> (<a href="https://cindicator.com">Cindicator SOC</a>) 🇷🇺
+      <h2>Участники спринтов</h2>
+      <h4>Звезды символизируют количество спринтов, в которых участвовал специалист</h4>
```
Comment (yugoslavskiy) [inline index_ru.html]: я бы сказал "Звезды отражают количество спринтов, в которых участвовал специалист". Символизируют — сложнее

File: sprints/sprint_2_ru.html [hunk 21]
```diff
@@ -110,6 +110,66 @@ <h2>Incident Response</h2>
...
+      - 23 новых теста <a href="https://github.com/redcanaryco/atomic-red-team/pull/1431"> Atomic Red Team</a> и 7 тестов обновлены; <br>
+      - 24 новых <a href="https://github.com/TheHive-Project/Cortex-Analyzers/pull/991">The Hive</a> Модуля Реагирования, включая Palo Alto NGFW, Duo Security, Gmail и Azure Active Directory; <br>
+      - <a href="https://github.com/SigmaHQ/sigma/pull/1158">Pull Request</a> от <a href="https://twitter.com/otr_community">OTR Community</a>. <br>
```
Comment (yugoslavskiy) [inline sprints/sprint_2_ru.html]: я думаю что лучше сказать об этом в предложении с результатами по Sigma, определив общее количество добавленных правил с обоих PR

File: sprints/sprint_3.html [hunk 22]
```diff
@@ -0,0 +1,46 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>OSCD: Sprint #2 Main</title>
```
Comment (yugoslavskiy) [inline sprints/sprint_3.html]: копипаста. должно быть `OSCD: Sprint #3 Main`

File: sprints/sprint_3_ru.html [hunk 23]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>OSCD: Sprint #2 Main</title>
```
Comment (yugoslavskiy) [inline sprints/sprint_3_ru.html]: копипаста. должно быть `OSCD: Sprint #3 Main`

File: sprints/sprint_3_ru.html [hunk 24]
```diff
@@ -0,0 +1,46 @@
...
+      <br>
+      <center> <a href="../index.html">Домой</a> </center>
+      <br>
+      <center> <a href="sprint_3.html">EN</a> | <a href="sprint_3_ru.html">RU</a> </center>
+      <div>
+      <p align="center">
+        <img src="../img/oscd_logo_v2.gif" alt="OSCD">
+      </p>
+      </div>
+      <center>Информация скоро будет опубликована. Следите за нашим <a href="https://twitter.com/oscd_initiative">twitter</a>.<br></center>
```
Comment (yugoslavskiy) [inline sprints/sprint_3_ru.html]: я бы написал в одно предложение: `Информация скоро будет опубликована в нашем <a href="https://twitter.com/oscd_initiative">twitter</a>` (так проще)

File: sprints/sprint_3_ru.html [hunk 25]
```diff
@@ -0,0 +1,46 @@
...
+      <br>
+      <center> <a href="sprint_3.html">EN</a> | <a href="sprint_3_ru.html">RU</a> </center>
+      <br>
+      <center> <a href="../index.html">Назад</a> </center>
```
Comment (yugoslavskiy) [inline sprints/sprint_3_ru.html]: думаю лучше по аналогии с тем что сверху — "Домой". или везде заменить на "На главную"/"Главная". главное чтобы везде одинаково было короче

**Comment by yugoslavskiy** [comment]
поправь еще пожалуйста https://github.com/zinint/oscd/blob/e194b25cf145abd354524c65dde1565e6768cd9f/sprints/sprint_1_ru.html#L65

описанни -> описании

File: index_ru.html (hunk 19 above)
Comment (zinint) [reply index_ru.html]: изображение
Согласовались же на этом определении, даже дошли до вердикта "збс, красота" (:

File: index_ru.html (hunk 18 above)
Comment (zinint) [reply index_ru.html]: Поправил тут - https://github.com/oscd-initiative/oscd/pull/4/commits/4aae098ea0b4b9836fb6e576797771550c2d571f , но еще раз указывать Твиттер мне кажется излишне, ибо он на одну строчку Выше уже упомянут.

File: index_ru.html (hunk 20 above)
Comment (zinint) [reply index_ru.html]: Поправил тут - https://github.com/oscd-initiative/oscd/pull/4/commits/37d98dea381c9892fc9cec69bd741d69988677f2

File: sprints/sprint_2_ru.html (hunk 21 above)
Comment (zinint) [reply sprints/sprint_2_ru.html]: Поправил тут https://github.com/oscd-initiative/oscd/pull/4/commits/9e59ab8bb46e2f3dcb17160c355343107e9aec58 (RU) и тут https://github.com/oscd-initiative/oscd/pull/4/commits/49a463a8fdc189b71ef60c87d61ba4df77aaf29c (EN)

**Comment by zinint** [comment]
> поправь еще пожалуйста https://github.com/zinint/oscd/blob/e194b25cf145abd354524c65dde1565e6768cd9f/sprints/sprint_1_ru.html#L65
> 
> описанни -> описании

Поправил тут https://github.com/oscd-initiative/oscd/pull/4/commits/7f7148eaabd55a374224f6350dc062c76ac5121d , спасибо, что заметил.

Всё исправлено, большая просьба прокомментировать снова.

File: index_ru.html (hunk 19 above)
Comment (yugoslavskiy) [reply index_ru.html]: сорян, упустил этот момент. казалось что мы более детально прорабатывали вопрос разницы между сообществом и инициативой и от принципа на котором мы остановились можно было бы отойти после отдельного обсуждения. я как-то быстро откомментил в тот раз, не заметил

File: sprints/sprint_2.html [hunk 26]
```diff
@@ -108,6 +108,65 @@ <h2>Incident Response</h2>
       <br>
       <!-- The coverage was built upon <a href="https://github.com/oscd-initiative/Cortex-Analyzers" target="_blank">our fork</a> of repo with TheHive Analyzers and Responders, that we tagged with RE&CT IDs. -->
       That's why we've decided to focus on the TheHive Responders development this time. A detailed backlog on Incident Response could be found <a href="sprint_2_backlog.html" target="_blank">here</a>.
+      <br><br>
+      <h2>Results</h2>
+      - 242 new <a href="https://github.com/SigmaHQ/sigma/pull/1362">Sigma</a> rules added and 305 updated in OSCD's Pull Request and 45 new rules added in <a href="https://github.com/SigmaHQ/sigma/pull/1158">Pull Request</a> from <a href="https://twitter.com/otr_community">OTR Community</a>; <br>
```
Comment (yugoslavskiy) [inline sprints/sprint_2.html]: тут стоит ссылку на PR крепить к "Pull Request", иначе не очевидно. сейчас результат работы скрывается за словом Sigma, а такая ссылка встречалась по тексту ранее и вела к репо Sigma на GitHub

Comment (yugoslavskiy) [reply sprints/sprint_2.html]: Что думаешь об этой версии?

-  287 new Sigma rules were added in total. 242 added and 305 updated by OSCD participants ([Pull Request](https://github.com/SigmaHQ/sigma/pull/1362)). 45 rules added by [OTR Community](https://twitter.com/otr_community) ([Pull Request](https://github.com/SigmaHQ/sigma/pull/1158))

File: index_ru.html (hunk 19 above)
Comment (yugoslavskiy) [reply index_ru.html]: "Открытая международная инициатива специалистов по компьютерной безопасности, созданная для решения общих проблем и распространения знаний." — норм?
Если согласен, то я повсюду это поправлю (в статье, на сайте, в английской версии и тд)

File: sprints/sprint_2.html (hunk 26 above)
Comment (yugoslavskiy) [reply sprints/sprint_2.html]: Если тебе ок, то я переведу для русской версии

File: index_ru.html (hunk 19 above)
Comment (zinint) [reply index_ru.html]: Ок. Я поправил на сайте и в статье (RU), в статье (EN) с Антоном не забудьте.

File: sprints/sprint_2.html (hunk 26 above)
Comment (zinint) [reply sprints/sprint_2.html]: Обновил для обоих версий

File: sprints/sprint_2.html [hunk 27]
```diff
@@ -108,6 +108,65 @@ <h2>Incident Response</h2>
       <br>
       <!-- The coverage was built upon <a href="https://github.com/oscd-initiative/Cortex-Analyzers" target="_blank">our fork</a> of repo with TheHive Analyzers and Responders, that we tagged with RE&CT IDs. -->
       That's why we've decided to focus on the TheHive Responders development this time. A detailed backlog on Incident Response could be found <a href="sprint_2_backlog.html" target="_blank">here</a>.
+      <br><br>
+      <h2>Results</h2>
+      - 287 new Sigma rules were added in total. 242 added and 305 updated by OSCD participants <a href="https://github.com/SigmaHQ/sigma/pull/1362">(Pull Request)</a>. 45 rules added by <a href="https://twitter.com/otr_community">OTR Community</a> <a href="https://github.com/SigmaHQ/sigma/pull/1158">(Pull Request)</a>; <br>
+      - 23 new <a href="https://github.com/redcanaryco/atomic-red-team/pull/1431"> Atomic Red Team</a> tests and 3 new artifacts added, and 7 tests updated; <br>
```
Comment (yugoslavskiy) [inline sprints/sprint_2.html]: та же тема, не понятно это линк к репо или линк к результатам наших работ — пулл риквесту. я думаю если по образу и подобию инфы о сигме сделать то будет ок

File: sprints/sprint_2_ru.html [hunk 28]
```diff
@@ -110,6 +110,65 @@ <h2>Incident Response</h2>
       <br>
       <!-- The coverage was built upon <a href="https://github.com/oscd-initiative/Cortex-Analyzers" target="_blank">our fork</a> of repo with TheHive Analyzers and Responders, that we tagged with RE&CT IDs. -->
       Вот почему мы решили сосредоточиться на разработке Модулей Реагирования платформы TheHive. Детальный список задач по данному направлению можно найти <a href="sprint_2_backlog_ru.html" target="_blank">здесь</a>.
+      <br><br>
+      <h2>Результаты</h2>
+      - 287 новых правил Sigma. Участники OSCD разработали 242 новых правила и обновили 305 старых <a href="https://github.com/SigmaHQ/sigma/pull/1362">(Pull Request)</a>. 45 новых правил разработали <a href="https://twitter.com/otr_community">OTR Community</a> <a href="https://github.com/SigmaHQ/sigma/pull/1158">(Pull Request)</a>; <br>
+      - 23 новых теста <a href="https://github.com/redcanaryco/atomic-red-team/pull/1431"> Atomic Red Team</a> и 7 тестов обновлены; <br>
```
Comment (yugoslavskiy) [inline sprints/sprint_2_ru.html]: та же тема с русской версией. обсуждали в личке %)

File: sprints/sprint_2.html (hunk 27 above)
Comment (zinint) [reply sprints/sprint_2.html]: Внес правки

File: sprints/sprint_2_ru.html (hunk 28 above)
Comment (zinint) [reply sprints/sprint_2_ru.html]: Внес правки

File: sprints/sprint_2.html [hunk 29]
```diff
@@ -108,6 +108,65 @@ <h2>Incident Response</h2>
       <br>
       <!-- The coverage was built upon <a href="https://github.com/oscd-initiative/Cortex-Analyzers" target="_blank">our fork</a> of repo with TheHive Analyzers and Responders, that we tagged with RE&CT IDs. -->
       That's why we've decided to focus on the TheHive Responders development this time. A detailed backlog on Incident Response could be found <a href="sprint_2_backlog.html" target="_blank">here</a>.
+      <br><br>
+      <h2>Results</h2>
+      - 287 new Sigma rules were added in total. 242 added and 305 updated by OSCD participants <a href="https://github.com/SigmaHQ/sigma/pull/1362">(Pull Request)</a>. 45 rules added by <a href="https://twitter.com/otr_community">OTR Community</a> <a href="https://github.com/SigmaHQ/sigma/pull/1158">(Pull Request)</a>; <br>
+      - 23 new Atomic Red Team tests, 3 new artifacts, and 7 updated tests <a href="https://github.com/redcanaryco/atomic-red-team/pull/1431">(Pull Request)</a>; <br>
+      - 24 new The Hive responders, including Palo Alto NGFW, Duo Security, Gmail and Azure Active Directory <a href="https://github.com/TheHive-Project/Cortex-Analyzers/pull/991">(Pull Request)</a>. <br>
```
Comment (yugoslavskiy) [inline sprints/sprint_2.html]: TheHive should be written in one word:
https://t.me/phd_soc/2748

Comment (yugoslavskiy) [reply sprints/sprint_2.html]: changed it myself 🙏

**Pull request merged**